dotCMS/src/main/java/com/dotmarketing/webdav/DotWebdavHelper.java

```diff
@@ -1015,6 +1026,17 @@ public void setResourceContent(String resourceUri, InputStream content,	String c
 				//Wiping out the thumbnails and resized versions
 				//http://jira.dotmarketing.net/browse/DOTCMS-5911
 				APILocator.getFileAssetAPI().cleanThumbnailsFromFileAsset(destinationFile);
+				
+				//Wipe out empty versions that Finder creates
+				List<Contentlet> versions = APILocator.getContentletAPI().findAllVersions(identifier, user, false);
```

Comment:
@erickgonzalez let's do this. Since we are already here , let's declare `ContentAPI conAPI = APILocator....` at the top of this class (~ line 108) replace all occurrences of 'APILocator.getContentletAPI()' with that instance var